tools/test.py

```diff
@@ -1303,24 +1321,35 @@ def ProcessOptions(options):
       return False
   if options.J:
     options.j = multiprocessing.cpu_count()
+  def CheckTestMode(name, option):
+    if not option in ["run", "skip", "dontcare"]:
+      print "Unknown %s mode %s" % (name, option)
+      return False
+    return True
+  if not CheckTestMode("--flaky-tests", options.flaky_tests):
+    return False
   return True
 
 
 REPORT_TEMPLATE = """\
 Total: %(total)i tests
  * %(skipped)4d tests will be skipped
+ * %(nocrash)4d tests are expected to be flaky but not crash
  * %(pass)4d tests are expected to pass
  * %(fail_ok)4d tests are expected to fail that we won't fix
  * %(fail)4d tests are expected to fail that we should fix\
 """
 
 def PrintReport(cases):
+  def IsFlaky(o):
+    return (PASS in o) and (FAIL in o) and (not CRASH in o) and (not OKAY in o)
```

Comment:
Umm.. this is old logic that I didn't write (https://github.com/joyent/node/commit/3fed1a095434877f6abe946ff6d788410b1dc66b). I guess I ported it because it was removed as part of the removal of flaky tests in io.js. I don't we think we depend on it in anyway. I will remove it.